Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/AWSCredentialIdentityResolvers/SSOAWSCredentialIdentityResolver.swift

```diff
@@ -5,15 +5,19 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
Uses `SSOBearerTokenIdentityResolver` to fetch SSO token and makes request to `SSO::getRoleCredentials` with it to fetch temporary AWS credentials. The SSO client used here is the internal client, instantiated and passed off to the resolver here from public service clients' auth scheme resolvers.